config.py

```diff
@@ -77,7 +78,11 @@ class DevelopmentConfig(Config):
     ASSETS_DEBUG = True
     SQLALCHEMY_DATABASE_URI = os.environ.get('DEV_DATABASE_URL') or \
         'sqlite:///' + os.path.join(basedir, 'data-dev.sqlite')
-    print('THIS APP IS IN DEBUG MODE. YOU SHOULD NOT SEE THIS IN PRODUCTION.')
+
+
+    @classmethod
```

Comment:
Also does @classmethod help fix the issue or is it just a semantic/stylistic change?

Reply:
Please see previous comment.